src/main/java/teammates/ui/controller/InstructorFeedbackTemplateQuestionAddAction.java

```diff
@@ -0,0 +1,77 @@
+import teammates.common.util.Const;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.StatusMessage;
+import teammates.common.util.StatusMessageColor;
+import teammates.ui.pagedata.PageData;
+
+public class InstructorFeedbackTemplateQuestionAddAction extends Action {
+
+    @Override
+    protected ActionResult execute() {
+        String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        InstructorAttributes instructorDetailForCourse = logic.getInstructorForGoogleId(courseId, account.googleId);
+
+        gateKeeper.verifyAccessible(instructorDetailForCourse,
+                logic.getFeedbackSession(feedbackSessionName, courseId),
+                false, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+
+        RedirectResult redirectResult = createRedirectResult(new PageData(account, sessionToken)
+                .getInstructorFeedbackEditLink(courseId, feedbackSessionName));
+
+        String[] feedbackQuestionNumbers = getRequestParamValues(Const.ParamsNames.FEEDBACK_QUESTION_TEMPLATE_NUMBER);
+
+        if (feedbackQuestionNumbers == null) {
+            statusToAdmin = "";
+            statusToUser.add(new StatusMessage("No template questions are indicated to be added",
+                    StatusMessageColor.DANGER));
+            isError = true;
+
+            return redirectResult;
+        }
+
+        List<FeedbackQuestionAttributes> templateQuestions = logic.populateFeedbackSessionTemplateQuestions("TEAMEVALUATION",
+                courseId, feedbackSessionName, account.getEmail());
+
+        try {
+            for (String questionNumber : feedbackQuestionNumbers) {
+
+                int questionNum = Integer.parseInt(questionNumber);
+                // there are only 5 template questions being populated at the moment
+                Assumption.assertTrue("Invalid question number", questionNum >= 1 && questionNum <= 5);
+
+                FeedbackQuestionAttributes feedbackQuestion = templateQuestions.get(questionNum - 1);
+                feedbackQuestion.questionNumber = -1;
+                logic.createFeedbackQuestion(feedbackQuestion);
+
+                statusToAdmin = "Added Feedback Template Question for Feedback Session:<span class=\"bold\">("
+                        + feedbackQuestion.feedbackSessionName + ")</span> for Course <span class=\"bold\">["
+                        + feedbackQuestion.courseId + "]</span> created.<br>"
+                        + "<span class=\"bold\">"
+                        + feedbackQuestion.getQuestionDetails().getQuestionTypeDisplayName()
+                        + ":</span> "
+                        + SanitizationHelper.sanitizeForHtml(feedbackQuestion.getQuestionDetails().getQuestionText());
+            }
+
+            statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_QUESTION_ADDED,
```

Comment:
Currently it is `The question has been added to this feedback session.` even when multiple questions were checked and added. Update statusToUser accordingly